server/src/main/resources/extensibles.json

```diff
@@ -0,0 +1,6 @@
+{
+  "org.elasticsearch.plugin.analysis.api.AnalyzerFactory":"org.elasticsearch.plugin.analysis.api.AnalyzerFactory",
+  "org.elasticsearch.plugin.analysis.api.CharFilterFactory":"org.elasticsearch.plugin.analysis.api.CharFilterFactory",
+  "org.elasticsearch.plugin.analysis.api.TokenFilterFactory":"org.elasticsearch.plugin.analysis.api.TokenFilterFactory",
+  "org.elasticsearch.plugin.analysis.api.TokenizerFactory":"org.elasticsearch.plugin.analysis.api.TokenizerFactory"
+}
```

Comment:
Maybe locate this resource file in a directory structure that closer resembles its usage, say `org/elasticsearch/plugins/scanner` ?